balancer-labs/frontend-v2
# Bugfix: Reload page on dynamic module load error

# Description

Workaround for [Failed to fetch dynamically imported module error.](https://github.com/vitejs/vite/issues/11804).

Closes #2778

Sentry link: https://balancer-labs.sentry.io/issues/3922269920/?project=5725878

## Type of change


## How should this be tested?

1. Open https://beta.balancer.fi/#/
2. Wait for merge and deploy and check if the error is gone. 

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
